internal/syscallcompat/sys_linux.go

```diff
@@ -54,9 +54,19 @@ func Renameat(olddirfd int, oldpath string, newdirfd int, newpath string) (err e
 	return syscall.Renameat(olddirfd, oldpath, newdirfd, newpath)
 }
 
-// Unlinkat wraps the Unlinkat syscall.
-func Unlinkat(dirfd int, path string) error {
-	return syscall.Unlinkat(dirfd, path)
+// Unlinkat syscall. In old versions the 'flags' argument was missing, so
+// manually call it by using the corresponding syscall number.
+func Unlinkat(dirfd int, path string, flags int) (err error) {
```

Comment:
Should get a test

Reply:
The syscall itself? Or do you mean tests which use Rmdir in a FUSE mount?

Reply:
Yes the syscall itself.

The Go stdlib uses https://golang.org/src/syscall/mksyscall.pl to autogenerate the wrappers, and different prototypes for amd64 ( https://golang.org/src/syscall/syscall_linux_amd64.go ) and the rest ( https://golang.org/src/syscall/syscall_linux.go ). Looks hard enough to get right (on 32 bit and 64 bit!) to warrant a test

Reply:
(I will add a test for syscallat as well)